### DISCUSSION TITLE
CI Builds

### DISCUSSION THREAD

**Discussion by Michael21107** [scripthookvdotnet/scripthookvdotnet#1341] [Q&A] [answered]
is there a way to see all nightly builds in this repo? for logged in users ofc

**Comment by kagikn** [comment, marked as answer]
Logged in users can download nightly (CI) builds from artifacts built in GitHub Actions runs. I'll put the URL for (build) runs for the main branch.
https://github.com/scripthookvdotnet/scripthookvdotnet/actions/workflows/build.yml?query=branch%3Amain

**Reply by Michael21107** [reply]
so CI builds in the standalone repo are just gonna be copies from main branch actions runs?

**Reply by kagikn** [reply]
Yes, I'm planning to make the nightly repo hold (some of) copies of artifacts for the main branch, but with no expiration.

**Reply by Michael21107** [reply]
gotcha, when would they normally expire?

**Reply by kagikn** [reply]
90 days after generated, can't extend longer

**Reply by Michael21107** [reply]
gotcha, thanks
